web/docs/tutorials/dev-excuses-app/adding-operations.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Adding operations
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Now you'll need to create two files: `actions.js` and `queries.js` in the `ext` folder.
+
+Let’s add `saveExcuse()` action to our `actions.js` file. This action will save the text of our excuse to the database. 
+
+```js title=".../ext/actions.js | Defining an action"
+export const saveExcuse = async (excuse, context) => {
+  return context.entities.Excuse.create({
+    data: { text: excuse.text }
+  })
+}
+```
+
+Then we need to create two queries in the `queries.js` file. First, one `getExcuse` will call an external API and fetch a new excuse. The second one, named `getAllSavedExcuses`, will pull all the excuses we’ve saved to our database. 
+
+```js title=".../ext/queries.js | Defining queries"
+import axios from 'axios';
+
+export const getExcuse = async () => {
+  return axios
+    .get('https://api.devexcus.es/')
+    .then(res => {
+      return res.data;
+    })
+    .catch(error => {
+      console.error(error);
+    });
+}
```

Comment:
Maybe this is a bit nicer (using `await`).
```suggestion
export const getExcuse = async () => {
  try {
    const response = await axios.get('https://api.devexcus.es/')
    return response.data
  catch (error) {
    console.error(error)
  }
}
```

Reply:
Also, it is probably better if we don't catch any errors so they get rethrown and handled up one the client. I see that we anyway catch them again when we call the function in the React component (MainPage). Otherwise, with current approach, you print errors on the server console, but client never gets an error because you don't rethrow it.

So just
```js
export const getExcuse = async () => {
  const response = await axios.get('https://api.devexcus.es/')
  return response.data
}
```